### DISCUSSION TITLE
Show the date format as a placeholder in the Date input field

### DISCUSSION THREAD

**Discussion by rungta** [craftcms/cms#12284] [Ideas] [answered]
A date field without a value currently only shows an icon and a blank input. It has frequently lead to clients assuming they have to use the date picker UI to enter a date. Can we somehow also inform users that the field allows a date to be typed in as well, and inform them of the expected format (based on the locale)?

We recently did this on a project by hooking into the [DefineFieldHtmlEvent](https://docs.craftcms.com/api/v3/craft-events-definefieldhtmlevent.html), but it might be good to have this as native behaviour.

![image](https://user-images.githubusercontent.com/663677/200850805-c7819f34-ec32-47d9-ab74-483d88b5e83d.png)

**Comment by brandonkelly** [comment, marked as answer]
Open to this, but we’d need to do it for time inputs as well for consistency, and even with the icons removed, there’s not enough space in Post/Expiry Date inputs to fit the whole time format as placeholder text:

<img width="364" alt="An entry’s Post Date and Expiry Date fields, showing that the “HH:MM AM/PM” placeholder text is truncated." src="https://user-images.githubusercontent.com/47792/202008090-9a521777-1c22-482a-bf90-2cbd37b54881.png">

So we’ll need to hold off on this until Date + Time inputs get redesigned.
